templates/repository-template/README.md

```diff
@@ -1,23 +1,35 @@
-# Project Title
+> **NOTE:** It is a general template that can be used for a project README.md, example README.md, or any other README.md type in all Kyma repositories in the Kyma organization. Not all the sections are mandatory. Use only those that suit your use case but keep the proposed section order.
 
-Modify the project title and insert the name of your project. Use Heading 1 (H1).
+# {Project Title}
+
+> Modify the title and insert the name of your project. Use Heading 1 (H1).
 
 ## Overview
 
-Provide a description of the project's functionality.
+> Provide a description of the project's functionality.
+>
+> If it is an example README.md, describe what the example illustrates.
 
 ## Prerequisites
 
-List the requirements to run the project.
+> List the requirements to run the project or example.
 
 ## Installation
 
-Explain the steps to install your project. Include the instructions or provide links to the related documentation.
+> Explain the steps to install your project.
```

Comment:
I'd add the information that the steps should be a numbered list (or more than one, if there are several separate tasks to do). It's implied in the word "steps" but maybe readers think of subheadlines etc instead (whereas I think subheadlines should outline the structure if there are more than one set of steps).

Reply:
OK, good point Nina :)